_episodes/15-practices.md

```diff
@@ -22,7 +22,47 @@ no different from drama, music, or athletics.
 And as in those fields,
 we have a collection of small tips and tricks to make teaching work better.
 
-### Challenges
+## The Code of Conduct
+
+Software Carpentry and Data Carpentry have a shared code of conduct
+(online [here]({{ site.swc_site }}/conduct/) and [here]({{ site.dc_site }}/code-of-conduct/))
+and participants in our workshops are required to abide by it.
+Hosts *must* point people at it during registration,
+and instructors *must* remind attendees of it at the start of the workshop.
+The Code of Conduct doesn't just tell everyone what the rules are:
+it tells them that there *are* rules,
+and that they can therefore expect a comfortable learning experience.
+
+What's the purpose of a Code of Conduct?
+:   A Code of Conduct cannot stop people from being offensive, any more
+    than laws against theft stop people from stealing.  What the CoC
+    *can* do is make expectations and consequences clear.
+
+Do workshop participants ever actually violate the Code of Conduct?
+:   Very rarely, but it has happened.
+
+But what about free speech?
+:   People are free to say what they want, but that doesn't mean they
+    are free to say it in our workshops---particularly not if it is
+    making someone else feel uncomfortable or unwelcome.
+
+What should I do if someone violates the Code of Conduct?
+:   If you are an instructor, and believe that someone in a workshop has
+    violated the Code of Conduct, you may warn them, ask them to
+    apologize, and/or expel them, depending on the severity of the
+    violation and whether or not you believe it was intentional.  No
+    matter what you choose to do, you should contact
```

Comment:
To make clear the purpose of informing SWC admin, maybe add what SWC admin will do with such emails?. What if this happens at a Data Carpentry workshop, does one also contact SWC admin in those cases?